src/xdocs/config_annotation.xml

```diff
@@ -738,7 +738,32 @@ public static final int COUNTER = 10; // violation
         <source>
 &lt;module name=&quot;MissingOverride&quot;/&gt;
         </source>
+          <p>Example:</p>
+          <source>
```

Comment:
Please fix indentation for 2 lines above - it should not be indented from previous tag, they should be on same indentation level.

Reply:
ok, done